src/services/analytics.ts

```diff
@@ -0,0 +1,33 @@
+interface GAPayload {
+  action: string
+  category?: string
+  label: string
+  value?: string
+}
+
+interface GAEventPayload {
+  eventAction: string
+  eventCategory: string
+  eventLabel: string
+  eventValue: string
+  hitType: string
+}
+
+const w: any = window
```

Comment:
if we move this declaration down into the `sendGA` function we don't need to mock any of the tests that aren't touched by the analytics code, which is preferable.